Content.Shared/Radio/Components/EncryptionKeyHolderComponent.cs

```diff
@@ -42,6 +42,13 @@ public sealed partial class EncryptionKeyHolderComponent : Component
     public Container KeyContainer = default!;
     public const string KeyContainerName = "key_slots";
 
+    /// <summary>
+    ///     Whether or not the headset can be examined to see the encryption keys while the keys aren't accessible.
```

Comment:
Huge shame this isn't a generic component that lets you block examining keys entirely.